university-college-lillebaelt-vancouver.csl

```diff
@@ -270,7 +278,7 @@
   </citation>
   <bibliography et-al-min="7" et-al-use-first="6" second-field-align="flush">
     <layout>
-      <text variable="citation-number" suffix="."/>
+      <text variable="citation-number" suffix=". "/>
```

Comment:
yes -- or am I missing something? I though if anything the space could mess with things once you get into higher numbers (i.e. 3 digits)